.github/workflows/release.yaml

```diff
@@ -7,48 +7,47 @@ jobs:
   publish:
     strategy:
       matrix:
-        os: [ ubuntu-20.04 ]
+        os: [ubuntu-20.04]
     runs-on: ${{ matrix.os }}
     steps:
-    - name: 📥 Checkout repository
-      uses: actions/checkout@v3
-      with:
-        # Fetch all history for all tags and branches
-        fetch-depth: 0
-        ref: master
-        token: ${{ secrets.BOT_GH_TOKEN }}
+      - name: 📥 Checkout repository
+        uses: actions/checkout@v3
+        with:
+          # Fetch all history for all tags and branches
+          fetch-depth: 0
+          ref: master
+          token: ${{ secrets.BOT_GH_TOKEN }}
 
-    - name: 🧰 Setup Node.js
-      uses: actions/setup-node@v1
-      with:
-        node-version: 18.12.0
-        registry-url: 'https://registry.npmjs.org'
+      - name: 🧰 Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 18.12.0
+          registry-url: 'https://registry.npmjs.org'
 
-    - name: Import bot's GPG key for signing commits
-      id: import-gpg
-      uses: crazy-max/ghaction-import-gpg@v5
-      with:
-        gpg_private_key: ${{ secrets.BOT_GPG_KEY }}
-        passphrase: ${{ secrets.BOT_GPG_PASSPHRASE }}
```

Comment:
I guess this is the only relevant change? It feels weird to have gpg key without a passphrase but I guess it does not make much difference

Reply:
Both secrets are held by the GH action and the same GH secret store, so a passphrase only adds complexity at no gain in security.